src/seedsigner/models/decode_qr.py

```diff
@@ -9,14 +9,13 @@
 from pyzbar import pyzbar
 from pyzbar.pyzbar import ZBarSymbol
 from urtypes.crypto import PSBT as UR_PSBT
-from urtypes.crypto import Account, HDKey, Output, Keypath, PathComponent, SCRIPT_EXPRESSION_TAG_MAP
+from urtypes.crypto import Account, Output
 from urtypes.bytes import Bytes
 
 from seedsigner.helpers.ur2.ur_decoder import URDecoder
-from seedsigner.models.psbt_parser import PSBTParser
-
-from . import QRType, Seed
-from .settings import SettingsConstants
+from seedsigner.models import QRType, Seed
+from seedsigner.models.settings import SettingsConstants
+from seedsigner.models.settings_definition import SettingsDefinition
```

Comment:
I believe this is unused.

Reply:
Affirmative! Cleaned up a few others as well.